### PR TITLE
feat(python): Update

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Fixed

**Steps to consider while deploying**
- Configuration changes:
- Documentation updates:
- Inform users by email:

### Review:
- [ ] Code approved by
- [ ] Tests executed on stage by:  (Document the test done with screen shots and description.)
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
### API 👍 
Screenshot 2024-03-07 at 11 29 27

**Review by ChrOertlin — APPROVED** [review]
Looks good!

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-FROM python:3.10-slim-bullseye
+FROM python:3.11-slim-bullseye
```
Comment (ChrOertlin) [inline Dockerfile:1]:
```suggestion
FROM docker.io/library/python:3.11-slim-bullseye as base
```
perhaps?

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-LABEL base_image="python:3.10-slim"
+LABEL base_image="python:3.11-slim"
```
Comment (ChrOertlin) [inline Dockerfile:3]:
```suggestion
```
with above

**Comment by henrikstranneheim** [comment]
Deployed to prod.

**Pull request merged**